packages/slo-stat-bot/src/slo-bot.ts

```diff
@@ -99,6 +121,34 @@ async function getSloFile(
   return sloRules;
 }
 
+/**
+ * Function gets list of open issues from the repository
+ * @param github unique installation id for each function
+ * @param owner of issue or pr
+ * @param repo of issue or pr
+ * @returns IssueListForRepoItem which contains issue number, user login, labels, assignees, issue created time, & issue updated time
+ */
+async function getIssueList(
+  github: GitHubAPI,
+  owner: string,
+  repo: string
+): Promise<IssueListForRepoItem[] | null> {
+  try {
+    const state = 'open';
+    const issueList = await github.issues.listForRepo({
+      owner,
+      repo,
+      state,
+    });
+    return issueList.data;
+  } catch (err) {
+    console.error(
```

Comment:
A few things here!
- For all logging, we're now using `logger.error`, where `logger` comes from `import {logger} from 'gcf-utils';`
- When dealing with errors, it's import you both log the error, *and* it's call stack. The pattern we're starting to widely use here is:

```js
err.message = `Error in getting list of issues from repo ${repo}: ${err.message}`;
logger.error(err);
```

Logging the entire error object ensures all fields will be included and we use the right logging level in Cloud Logging.  Please do a double check and make sure this is happening broadly!

Reply:
Sorry that's my bad. Didn't catch that inconsistency but will fix it!